pandas/tests/test_lib.py

```diff
@@ -184,6 +185,52 @@ def test_get_reverse_indexer(self):
         self.assertTrue(np.array_equal(result, expected))
 
 
+class TestNullObj(tm.TestCase):
+
+    def test_basic(self):
+        arr = np.array([1, None, 'foo', -5.1, pd.NaT, np.nan])
+        expected = np.array([False, True, False, False, True, True])
+
+        result = lib.isnullobj(arr)
+        tm.assert_numpy_array_equal(result, expected)
+
+    def test_non_ndarray_inp(self):
+        ind = pd.Index([1, None, 'foo', -5.1, pd.NaT, np.nan])
```

Comment:
The specific `lib` functions aren't being tested anywhere, so I intended to put unit tests for them in `test_lib.py`.  The `null` functions you were referring to are those exposed at the Python layer, not the internal ones (which is what I am fixing) that are written in Cython.